lib/order.rb

```diff
@@ -0,0 +1,37 @@
+require_relative 'menu'
+require_relative 'text'
+
+class Order
+  attr_accessor :menu, :order, :total, :text
+  def initialize
+    @order = []
+    @total = 0
+    @menu = Menu::MENU
+    @text = TextConfirm.new
+  end
+
+  def add(dish, amount = 1)
+    raise "#{dish} is not available" unless dish?(dish)
+
+    amount.times { @order << dish }
+    amount.times { @total += @menu[dish] }
+  end
+
+  def total_correct?
+    balance = 0
+    @order.map do |dish|
+      balance += @menu[dish]
+    end
+    @total == balance
```

Comment:
Might be nice to format this so it reads more like a price. at the moment it just shows the cost with no currency or decimal places.